rtgui/options.cc

```diff
@@ -2229,10 +2229,13 @@ int Options::saveToFile (Glib::ustring fname)
     FILE *f = g_fopen (fname.c_str (), "wt");
 
     if (f == nullptr) {
-        if (options.rtSettings.verbose) {
-            printf ("Options::saveToFile / Error: unable to open file \"%s\" with write access!\n", fname.c_str());
-        }
-
+        std::cout << "Warning! Unable to save your preferences to: " << fname << std::endl;
+#ifndef RAWTHERAPEE_CLI
+        Glib::ustring msg_ = Glib::ustring::compose(M("MAIN_MSG_WRITEFAILED"), fname.c_str());
```

Comment:
I think it would be better to check the return code of `Options::saveToFile` and let the callee decide what to do (i.e. show a dialog message) instead of unconditionally show a dialog in here